tests/test_dask_image/test_ndmeasure/test_core.py

```diff
@@ -110,7 +112,12 @@ def test_center_of_mass(datatype):
     ]
 )
 def test_measure_props(funcname, shape, chunks, has_lbls, ind):
-    sp_func = getattr(spnd, funcname)
+    if funcname == 'sum_labels' and scipy.__version__ > LooseVersion('1.5.0'):
```

Comment:
```suggestion
    if funcname == 'sum_labels' and scipy.__version__ < LooseVersion('1.5.0'):
```

If scipy version is old, use function name `scipy.ndimage.sum`
But if scipy version is new, we can use the new `scipy.ndimage.sum_labels`

Reply:
Sorry if I have made this confusing. Please add a comment if you think that would make things less confusing for other people 😄

Reply:
+1 for comment. I confused 😁 This is the place which we will need a comment 👍